### PR TITLE
Lottery homework fix 3.

### PR DESCRIPTION
**Pull request by Stim1**
Added new commit with fixed mistakes.

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
+    public static final int BALL = 3;
```
Comment (aaananas) [inline src/main/java/core/basesyntax/Application.java:5]:
```suggestion
    public static final int BALL_NUMBER = 3;
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int BALL_SUPPLIER = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+
```
Comment (aaananas) [inline src/main/java/core/basesyntax/Lottery.java:11]:
```suggestion
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int BALL_SUPPLIER = 100;
```
Comment (aaananas) [inline src/main/java/core/basesyntax/Lottery.java:6]:
```suggestion
    public static final int MAX_NUMBER = 100;
```
lets choose better naming

**Review by aaananas — CHANGES_REQUESTED** [review]
leave this homework in this branch, but for future remember to create a new branch when you start doing your homework
Great job, just a few adjustments

**Comment by Stim1** [comment]
Added new commit with fixed mistakes.

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 4]
```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int colorNumber = new Random().nextInt(Color.values().length);
```
Comment (zachar-chorny) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]:
```suggestion
    Random random = new Random();

    public Color getRandomColor() {
        int colorNumber = random.nextInt(Color.values().length);
```
Random must be a class variable

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 5]
```diff
@@ -1,7 +1,10 @@
...
+        return Color.valueOf(Color.values()[colorNumber].name());
```
Comment (zachar-chorny) [inline src/main/java/core/basesyntax/ColorSupplier.java:8]:
```suggestion
        return Color.values()[colorNumber]
```
You can make it easier

**Review by zachar-chorny — CHANGES_REQUESTED** [review]

**Pull request closed without merging**